src/spectrecoff-cli/commands/Dumpify.fs

```diff
@@ -0,0 +1,51 @@
+namespace SpectreCoff.Cli.Commands
+
+open Dumpify
+open Spectre.Console.Cli
+open SpectreCoff
+open SpectreCoff.Dumpify
+
+type DumpifySettings() =
+    inherit CommandSettings()
+
+type DumpifyExample() =
+    inherit Command<DumpifySettings>()
+    interface ICommandLimiter<DumpifySettings>
+
+    override _.Execute(_context, _settings) =
+        // use the dump function to print all kinds of objects
+        let list = [|1;2;3|]
+        dump list |> ignore
+        {| Diameter = 5.5; Area = 3.3; Circumference = 2.2 |} |> dump |> ignore
+
+        // use the customDump function to fine-tune the parameters for your dumping
+        let colorConfig = ColorConfig()
```

Comment:
What is `ColorConfig()`?

Reply:
It's one of the configuration objects that can be passed to the `Dump()` method for customization. I chose it because it is easy to use and visible in the output (note the red color on the second Dump of the screenshot :))

https://github.com/MoaidHathot/Dumpify/blob/main/src/Dumpify/Config/ColorConfig.cs